modular_skyrat/modules/loadouts/loadout_items/loadout_datum_masks.dm

```diff
@@ -97,6 +97,9 @@ GLOBAL_LIST_INIT(loadout_masks, generate_loadout_items(/datum/loadout_item/mask)
 /*
 *	MISC
 */
+/datum/loadout_item/mask/surgical //I'm putting this closer to where it used to be because people WILL just assume its gone otherwise
+	name = "Sterile Mask"
```

Comment:
```suggestion
	name = "Recolorable Sterile Mask"
```
Always better to specify this, especially when its a change to an item that didn't used to have it
Yes, there IS an icon beside it. People will still get confused. Somehow.